util/reflect.go

```diff
@@ -1053,6 +1057,10 @@ func getNodesList(schema *yang.Entry, root interface{}, path *gpb.Path) ([]inter
 						// If the key is not filled, it is assumed to match.
 						continue
 					}
+					// NOTE: Normally we'd like to use ygot.KeyValueAsString for conversion
```

Comment:
I don't think we have to repeat the whole comment here, perhaps just say "as above, we don't require this to be a YANG enum string representation".